Staticfile

```diff
@@ -1 +1,2 @@
 root: public
+directory: visible
```

Comment:
why does directory listing need to be on?

Reply:
It isn't needed but is helpful viewing/copying full urls for development purposes.

Reply:
It would be nicer if this was enabled only in dev but I guess it is fine.

Reply:
If this is a useful feature to enable during development then simply enable it on your feature branch while working. Since this kind of change should not get promoted, it shouldn't be cemented in the git history for this file.

We _could_ start splitting the config and create a Staticfile-<dev|test|prod> but since the idea is to have as little drift as possible between environments I would really prefer we didn't go down that path.

Reply:
So tldr; remove this line